src/claimable/createUserPublish.ts

```diff
@@ -39,111 +39,101 @@ export async function publishToClaimableAccount(releaseId: string) {
-      let encodedUserId = await userRepo.match(source.ddexKey, [artistName])
-      if (!encodedUserId) {
-        // no user: create claimable user
-        console.log(`=== creating claimable account for ${artistName}`)
-        const hedgehog = getHedgehog()
-        const identityResult = await hedgehog.signUp({
-          username: email,
-          password,
-        })
-        console.log('identityResult', identityResult)
-
-        const audiusWalletClient = createHedgehogWalletClient(getHedgehog())
-        const userSdk = sdk({
-          appName: 'ddex',
-          environment: source.env || 'staging',
-          services: {
-            audiusWalletClient,
-          },
-        })
-
-        const discoveryResult = await userSdk.users.createUser({
-          metadata: {
-            handle: handle,
-            name: artistName,
-            wallet: identityResult.getAddressString(),
-            bio: `${release.labelName}.`,
-          },
-        })
-
-        await publogRepo.log({
-          release_id: release.key,
-          msg: 'created user',
-          extra: discoveryResult,
-        })
-
-        // const entropy = localStorage.getItem('hedgehog-entropy-key')
-        encodedUserId = encodeId(discoveryResult.metadata.userId)
-        console.log(discoveryResult, encodedUserId)
-
-        // upload profile picture + cover photo
-        const updateImageResult = await userSdk.users.updateProfile({
-          userId: encodedUserId,
-          metadata: {},
-          profilePictureFile: imageFile as any,
-          coverArtFile: imageFile as any,
-        })
-        console.log('updateImageResult', updateImageResult)
-
-        await publogRepo.log({
-          release_id: release.key,
-          msg: 'set user image',
-          extra: updateImageResult,
-        })
-
-        // authorize source ddex app
-        const grantResult = await userSdk.grants.createGrant({
-          userId: encodedUserId,
-          appApiKey: source.ddexKey,
-        })
-        console.log('grantResult', grantResult)
-
-        await publogRepo.log({
-          release_id: release.key,
-          msg: 'user grant',
-          extra: grantResult,
-        })
-
-        // save user details to db
-        await userRepo.upsert({
-          id: encodedUserId,
-          apiKey: source.ddexKey,
+  const email = `ddex-support+${handle}@audius.co`
+  const password = randomBytes(16).toString('hex')
+
+  try {
+    // attempt to find existing user record
+    // if not found, create a claimable account
+    let encodedUserId = await userRepo.match(source.ddexKey, [artistName])
+    if (!encodedUserId) {
+      // no user: create claimable user
+      console.log(`=== creating claimable account for ${artistName}`)
+      const hedgehog = getHedgehog()
+      const identityResult = await hedgehog.signUp({
+        username: email,
+        password,
+      })
+      console.log('identityResult', identityResult)
```

Comment:
Did you mean to leave this in? We made efforts in the past to avoid sending this stuff to axiom